packages/react-scripts/template/README.md

```diff
@@ -729,6 +730,32 @@ Please be advised that this is also a custom feature of Webpack.
 **It is not required for React** but many people enjoy it (and React Native uses a similar mechanism for images).<br>
 An alternative way of handling static assets is described in the next section.
 
+## Adding GraphQL files
+
```

Comment:
This needs to have a note about it being supported only in 2.0.0+, like other similar features.

Reply:
Done ✅